### PR TITLE
Begin

### PR DESCRIPTION
**Pull request by miguelfrde**
Please review:
- [x] @albertein 
- [x] @pablasso

### CODE REVIEW

**Comment by albertein** [comment]
Giphy

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+language: python
+
+sudo: false
+
+matrix:
+  include:
+    - python: 3.3
+      env:
+        - TASK=py33
+    - python: 3.4
```
Comment (pablasso) [inline .travis.yml:10]: Are we really using 3.3 and 3.4?

**Comment by pablasso** [comment]
:shipit:

File: .travis.yml (hunk 1 above)
Comment (miguelfrde) [inline .travis.yml:10]: Not us. But since this is being open sourced I wanted to support Python 3 at least from 3.3.

Comment (pablasso) [inline .travis.yml:10]: Got it, thanks.

**Pull request merged**